great-idea/css/index.css

```diff
@@ -105,6 +109,7 @@ nav a {
 .logo{
 	display:flex;
 	align-items: flex-end;
+	margin-left: -44px;
```

Comment:
Negative margins aren't wrong but aren't considered best practice. They can have a lot of unintended side effects. Be sure to watch for those unintended side effects.